etherman/etherman.go

```diff
@@ -41,6 +41,11 @@ import (
 	"golang.org/x/crypto/sha3"
 )
 
+const (
+	// ETrogUpgradeVersion is the version of the LxLy upgrade
+	ETrogUpgradeVersion = 2
```

Comment:
What `ETrogUpgradeVersion = 2` means?

Reply:
This the way that @invocamanman   suggested to discover the block: check the event `initialized` where  `version` is `ETrogUpgradeVersion`

Reply:
I guess this is the initialize version!?, i would even specify something like:
`initializeRollupManagerEtrogVersion` or something like that ^^

Reply:
Finally I change the implementation in favour of L1InfoTreeUpdate that have more sense